xml/System.Diagnostics.Tracing/EventSource.xml

```diff
@@ -2021,6 +2021,9 @@ public void Info1(string arg1)
 ```  
   
  You can also create a custom overload by using the <xref:System.Diagnostics.Tracing.EventSource.WriteEventCore%2A> method.  
+
+> [!IMPORTANT]
+> Event parameters with a type of `string` should not include `<` or `\0` characters. They are unsupported characters and can cause issues for parsers of the event payload.
```

Comment:
The limitation for angle brackets is in the name of the EventSource method, not the string parameters. Both '<' and '>' are problematic.